src/css/GrainResults.css

```diff
@@ -16,3 +16,18 @@
   width: 100%;
   padding-bottom: 15px;
 }
+
+.grains-search-error-msg {
```

Comment:
I appreciate your attention to details like this -- you sweat the small stuff, and the app is better for it.